### PR TITLE
rpcserver: show inactive channels in GetInfo

### LINKED ISSUE #1779: [Feature request] Show the number of inactive channels inf `GetInfo`
Currently a call to `getInfo` will show the number of pending and active channels:

```bash
# lncli --network=testnet getinfo
{
    "identity_pubkey": "xxx",
    "alias": "xxx",
    "num_pending_channels": 0,
    "num_active_channels": 18,
    "num_peers": 24,
    "block_height": 1408630,
    "block_hash": "0000000000000217d7d8affbffb25e50ce097fb4367f487ac8114abea065c7da",
    "synced_to_chain": true,
    "testnet": true,
    "chains": [
        "bitcoin"
    ],
    "uris": [],
    "best_header_timestamp": "1535105615",
    "version": "0.4.2-beta commit=c9bb04efb921eaa7700a376058bce7b8bdbb3df3"
}
```

It would be useful to also show the number of `inactive` channels.

The active channels are fetched here: https://github.com/lightningnetwork/lnd/blob/625b210f441ece841c76b81377dd96e8d09aba8e/rpcserver.go#L1243

### Steps to completion:
- [ ] Add a new field to the `GetInfoResponse` proto called `num_inactive_channels`: https://github.com/lightningnetwork/lnd/blob/625b210f441ece841c76b81377dd96e8d09aba8e/lnrpc/rpc.proto#L1040
- [ ] After getting the active channels, do a database lookup to get all currently open channels.
- [ ] Subtract the active channels from the number of open channels and add to the rpc response.

### PR DESCRIPTION
**Pull request by xsb**
Adds the `num_inactive_channels` field in the response of the `GetInfo` gRPC call. Inactive channels is calculated substracting the number of active channels to the sum of all currently open channels.

Fixes #1779

### CODE REVIEW

File: lnrpc/rpc.proto [hunk 1]
```diff
@@ -1051,11 +1051,14 @@ message GetInfoResponse {
     /// Number of active channels
     uint32 num_active_channels = 4 [json_name = "num_active_channels"];
 
+    /// Number of inactive channels
+    uint32 num_inactive_channels = 5 [json_name = "num_inactive_channels"];
```
Comment (halseth) [inline lnrpc/rpc.proto]: Proto definitions must keep their numbers for backwards compatibility. Instead `num_inactive_channels` should be defined with the first non-used number (looks like it is 15).

**Comment by xsb** [comment]
rebased

**Comment by halseth** [comment]
Needs a rebase! Otherwise LGTM 👍

File: lnrpc/rpc.swagger.json [hunk 2]
```diff
@@ -1724,6 +1724,11 @@
         "version": {
           "type": "string",
           "description": "/ The version of the LND software that the node is running."
+        },
```
Comment (halseth) [inline lnrpc/rpc.swagger.json]: this change should be squashed with the first commit.

**Comment by xsb** [comment]
@halseth rebased. I also squashed the commits you mentioned in the comment.

**Comment by xsb** [comment]
rebased again, rpc.pb.go was updated on master

**Pull request merged**